src/binaries/BinariesManager.ts

```diff
@@ -19,24 +18,16 @@ export class BinariesManager {
                                      accountContextManager: AccountContextManager,
                                      expectedCLIVersion: string
     ): IBinariesUtility {
-        if (this.useBinariesUtilityV2()) {
-            logger.trace(TelemetryEvent.BinariesUtility_VersionV2);
+        if (this.useBinariesUtility()) {
+            logger.trace(TelemetryEvent.BinariesUtility_Version);
             return new BinariesUtilityV2(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
         }
-
-        logger.trace(TelemetryEvent.BinariesUtility_VersionV1);
-        return new BinariesUtility(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
     }
 
-    private static useBinariesUtilityV2(): boolean {
+    private static useBinariesUtility(): boolean {
         const useBinaryUtilityEnvironmentVariable: string = process.env.BRIDGE_BINARYUTILITYVERSION;
-        if (useBinaryUtilityEnvironmentVariable != null) {
-            if (useBinaryUtilityEnvironmentVariable.toLowerCase() === `v1`) {
-                return false;
-            }
-            else if (useBinaryUtilityEnvironmentVariable.toLowerCase() === `v2`) {
-                return true;
-            }
+        if (useBinaryUtilityEnvironmentVariable != null && useBinaryUtilityEnvironmentVariable.toLowerCase() === `v2`) {
```

Comment:
I think by default we are planning to use v2 , we can remove this useBinariesUtility() method all together ? what do you think? 

ex: always  return new BinariesUtilityV2(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion); without the if condition check for useBinariesUtility.